File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/request/RequestShareItem.java
package app.editors.manager.mvp.models.request;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RequestShareItem {

    @SerializedName("ShareTo")
    @Expose
    private String shareTo;
    @SerializedName("Access")
    @Expose
    private String access;

    public String getShareTo() {
        return shareTo;
    }

    public void setShareTo(String shareTo) {
        this.shareTo = shareTo;
    }

    public String getAccess() {
        return access;
    }

    public void setAccess(String access) {
        this.access = access;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/request/RequestShareItem.kt
package app.documents.core.network.manager.models.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RequestShareItem(
    @SerializedName("ShareTo")
    @Expose
    var shareTo: String? = null,

    @SerializedName("Access")
    @Expose
    var access: String? = null
)